.travis.yml

```diff
@@ -10,9 +10,9 @@ cache:
     - $HOME/.gradle/wrapper/
 
 # required for oraclejdk9
-sudo: required
+sudo: false
 dist: trusty
-group: edge
+group: beta
```

Comment:
It's related to a container-based infrastructure (a separate [commit](70309c01ab5bc24596c3539fba26aee5b7db5377)) as itself was needed due to aforementioned issue with JAVA_HOME.